src/NetAppFiles/NetAppFiles.Test/NetAppFiles.Test.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>NetAppFiles</PsModuleName>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Az.Test.props" />
+
+  <PropertyGroup>
+    <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)$(AzTestAssemblySuffix).ScenarioTests</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.NetApp" Version="0.9.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.9.0-preview" />
```

Comment:
@leonardbf I believe the build is failing because the version of the Network SDK that you're using is now out of date with what is in the `master` branch -- you will want to pull the latest from the `master` branch and update this version to `19.10.0-preview`